addon-test-support/dom/-get-closest-dom-parent.js

```diff
@@ -0,0 +1,22 @@
+/*
+  element.closest() is not supported in IE11, so for backwards compatibility we use this polyfill from
+  https://developer.mozilla.org/en-US/docs/Web/API/Element/closest#Polyfill
+ */
+export function getClosestDomParent(element, selector) {
```

Comment:
I think this can _probably be omitted_ in favor of people providing the polyfill in their own test suites if they need it. I would be very surprised to hear that people are running their Ember test suites against IE11 (but please let me know if you are!). We currently don't ship an `includes` polyfill for IE11, for example, so if someone's testing it on IE11, they're likely polyfilling a few things anyway.

Reply:
Yeah, I was a bit unsure about this as well. But if we make new assertion helpers anyway, then this wouldn't be breaking anyhow.